playgrounds/astro/tsconfig.json

```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "astro/tsconfigs/strict"
+  "extends": "astro/tsconfigs/strict",
+  "compilerOptions": {
+    "module": "Node16",
+    "moduleResolution": "Node16"
+  }
```

Comment:
i would say leave this one out for now.

in astro v3, there is built-in support for `"bundler"` mode, so no extra configuration will be needed after we upgrade